src/components/navigation/nav.js

```diff
@@ -5,15 +5,18 @@ import './nav.css';
 function Nav() {
   return (
     <header>
-      <h1 className="h1">BookStore CMS</h1>
-      <nav>
-        <button type="button">
-          <Link className="button" to="/">Books</Link>
-        </button>
-        <button type="button">
-          <Link className="button" to="/categories">Categories</Link>
-        </button>
-      </nav>
+      <div className="just">
+        <h1 className="h1">BookStore CMS</h1>
+        <nav>
+          <button className="button-2" type="button">
+            <Link className="nav-button" to="/">Books</Link>
+          </button>
+          <button className="button-2" type="button">
+            <Link className="nav-button" to="/categories">Categories</Link>
+          </button>
+        </nav>
+      </div>
+      <div className="Mask" />
```

Comment:
- [ ] As the previous reviewer mentioned kindly make sure to add a login/profile icon to the navbar and align the items to the left so it looks exactly as the provided Design

![Screenshot_20230512_095958](https://github.com/Booze33/book-store/assets/25227022/b330c787-b8ef-4d15-bcea-cc7f5f3e901b)